services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/utils/MongoDbDeviceRegistryUtils.java

```diff
@@ -35,6 +35,14 @@
  */
 public final class MongoDbDeviceRegistryUtils {
 
+    /**
+     * The name of the JSON property containing the device identifier.
```

Comment:
is this field really used for the device identifier? Or is it used for the device data itself?